gammapy/spectrum/counts_spectrum.py

```diff
@@ -58,6 +57,14 @@ def __init__(self, counts, energy, livetime=None, backscal=1):
             self._livetime = livetime
         else:
             self._livetime = Quantity(0, 's')
+        if muoneff is not None:
+            self.muoneff = muoneff
+        else:
+            self.muoneff = 0
```

Comment:
I would not set the default value to 0 but rather leave the attribute as 'None'. Here and also for the zenith angle the value 0 hast a physical meaning but does not make sense as a default value. The same hold for the livetime actually. An elegant way to write this is

```
self._livetime = None if livetime is None else Quantity(livetime)
```